docs/APIRef.Expect.md

```diff
@@ -52,6 +53,13 @@ await expect(element(by.id('RandomJunk959'))).toNotExist();
 await expect(element(by.id('UniqueId204'))).toHaveText('I contain some text');
 ```
 
+### `toHaveLabel(label)`
+- Similar to `toHaveText(text)`, but searches by accessibilityLabel (iOS) or by contentDescription (Android)
```

Comment:
The two shouldn’t be linked in such nonchalant manner. On iOS, text and label can have multiple meanings, and they wouldn’t be similar in cases where users care about the accessibility of their application. The documentation should split those completely.

Reply:
You probably read much more out of the "similar" word that was meant there. But, I agree, I'll rewrite it.

Reply:
My issue is that some users l, particularly in RN ecosystem, are not familiar with iOS concepts, and learn half-truths because of imprecise text they see here and there. I’m sure it’s the same for Android. This is why I try to push for accuracy, so everyone gets the correct picture.